example_token.txt

```diff
@@ -0,0 +1 @@
+20345458237:ATYNBNAvCjdljvfdghaaNqDzbtyulOFGJOGAIY-4gfhgb8
```

Comment:
I don't think it is a good idea to keep such files in root folder. Looks like we are using it only for the sake of example.